building_with_meson.md

```diff
@@ -9,8 +9,14 @@ into a problem._
 
 **Install build tools:** Use one of:
 
-- `mamba create -f environment.yml
-- `pip install -r build_requirements.txt  # also make sure you have pkg-config and the usual system dependencies for NumPy`
+- `mamba env create -f environment.yml && mamba activate numpy-dev
```

Comment:
I'm pretty sure `mamba create` is preferred over `mamba env create` (and same for `conda`). E.g., see https://mamba.readthedocs.io/en/latest/user_guide/mamba.html#quickstart.

Reply:
I could not get `mamba create` to work with a `-f <file>`. Are you sure it works (the same for conda)?